### DISCUSSION TITLE
Solved: Shell commands are disappearing (when the jTab plugin is installed, too)

### DISCUSSION THREAD

**Discussion by dokmanc** [Taitava/obsidian-shellcommands#351] [Debugging and testing]
I'm losing all of my commands. The New Shell Command button is gone and every tab of the option page just shows the Documentation link.

![image](https://github.com/Taitava/obsidian-shellcommands/assets/955641/9ed21aa7-e953-429c-a7d0-4c8bb8464c15)

Uninstalling and reinstalling doesn't fix the issue. To correct the issue, I've started a new repo from scratch and rebuilt it slowly but it keeps on happening.

**Comment by dokmanc** [comment]
I strongly suspect that the following plug in causes Shell Plug In to have this issue:
https://github.com/davfive/obsidian-jtab

**Reply by Taitava** [reply]
Hi. So do your shell commands show up if you disable the jTab plugin?

Also, just to make sure, can you open up the SC plugin's `data.json` settings file and see if your shell commands are still in that file?

**Comment by dokmanc** [comment]
Thanks for the reply. 
Yes, the commands were still in the data.json file and disabling the jTab plug in solves the problem with Shell Commands. I  use so many plug-ins , I was bound to encounter a conflict.

**Reply by Taitava** [reply]
You are welcome. I installed now the jTab plugin and I can confirm that it does make the shell commands disappear in the settings for me too.

**Comment by Taitava** [comment]
# Solved

This problem has been solved in the _jTab Guitar Codeblocks_ plugin's version `1.2.3`. More information: https://github.com/davfive/obsidian-jtab/issues/15#issuecomment-1987386420
